pkg/testframework/console.go

```diff
@@ -147,3 +147,65 @@ func MustRemoveConsole(t *testing.T, client *Clientset) error {
 	}
 	return nil
 }
+
```

Comment:
So at this point `framework/console.go` is for `console` `operator` functions.  I think we should keep it that way (and perhaps even rename it to `console-operator.go` as its not for top level config stuff.  We should probably put these `funcs` in a `deployments.go` file.  I'm worried that if we get into testing the `console operator` config, then the `console` top level config`, then the `clusteroperator` `console`, then the various resources for the `console` itself, and everything is kept in one file.... we will cry. 😂

Reply:
> We should probably put these funcs in a deployments.go file

I would rather keep them in the `testframework` pkg, since all the functions are e2e test specific.

Reply:
Agree (just meant inside `/pkg/testframework/` keeping things tidy).